.gitignore

```diff
@@ -1,5 +1,6 @@
 assets/stylesheets/*.css
 assets/images/icons
+assets/images/govuk
```

Comment:
if you freshly clone the repo and npm install, these are added as untracked files so need to be gitignored